index.bs

```diff
@@ -3484,6 +3641,301 @@ partial dictionary MLOpSupportLimits {
     </div>
 </details>
 
+<div class="note">
+  <details open>
+    <summary>
+    The behavior of the {{MLGraphBuilder/sign()}} operation can be [EMULATED]
+    </summary>
+    <pre highlight="js">
+    function sign(builder, input, options) {
+      const zero = builder.constant(input.dataType, 0);
+      const positiveOne = builder.constant(input.dataType, 1);
+      const negativeOne = builder.constant(input.dataType, -1);
+
+      return builder.where(
+        builder.greater(input, zero),
+        positiveOne,
+        builder.where(
+          builder.lesser(input, zero),
+          negativeOne,
+          zero));
+    }
+    </pre>
+  </details>
+</div>
+
+
+### dequantizeLinear ### {#api-mlgraphbuilder-dequantizelinear}
+Dequantizes an integer tensor to floating point tensor using the scale and zero-point bias, where `output = (input - zeroPoint) * scale`. The *scale* and *zeroPoint* tensors can be smaller than the *input* tensor as they are [=blockwise broadcastable=].
+
+<script type=idl>
+partial interface MLGraphBuilder {
+  MLOperand dequantizeLinear(MLOperand input,
+                             MLOperand scale,
+                             MLOperand zeroPoint,
+                             optional MLOperatorOptions options = {});
+};
+
+dictionary MLQuantizationSupportLimits {
+  MLSupportLimits input;
+  MLSupportLimits scale;
+  MLSupportLimits zeroPoint;
+  MLSupportLimits output;
+};
+
+partial dictionary MLOpSupportLimits {
+  MLQuantizationSupportLimits dequantizeLinear;
+};
+</script>
+
+<div dfn-for="MLGraphBuilder/dequantizeLinear(input, scale, zeroPoint, options)" dfn-type=argument>
+    **Arguments:**
+        - <dfn>input</dfn>: an {{MLOperand}}. The input tensor.
+        - <dfn>scale</dfn>: an {{MLOperand}}. The scale tensor to multiply each input value by after adjusting by the zero point. It must be [=blockwise broadcastable=] with the input.
+        - <dfn>zeroPoint</dfn>: an {{MLOperand}}. The zero point tensor to subtract from each input value. It has the same [=MLOperand/shape=] as the scale.
+        - <dfn>options</dfn>: an {{MLOperatorOptions}}. Specifies the optional parameters of the operation.
+
+    **Returns:** an {{MLOperand}}. The output tensor that contains the dequantized values.
+</div>
+
+<table id=constraints-dequantizelinear class='data' link-for="MLGraphBuilder/dequantizeLinear(input, scale, zeroPoint, options)">
+  <caption>Constraints for {{MLGraphBuilder/dequantizeLinear()}}</caption>
+  <thead>
+    <tr>
+      <th>operand</th>
+      <th>[=/allowed data types=]</th>
+      <th>[=/allowed ranks=]</th>
+    </tr>
+  </thead>
+  <tr>
+    <td>{{input}}</td>
+    <td>{{MLOperandDataType/"uint8"}}, {{MLOperandDataType/"uint32"}}, {{MLOperandDataType/"int32"}}</td>
```

Comment:
I don't see an algorithm step referencing the allowed data types - I'd expect to see something like

1. If input’s dataType is not one of its allowed data types (according to this table), then throw a TypeError.

... and similar for scale and zeroPoint.